.github/styles/Google/WordList.yml

```diff
@@ -64,6 +69,7 @@ swap:
   open-source: open source
   overview screen: recents screen
```

Comment:
is this the name of the screen?

Reply:
Good catch! This may be a purely Google thing. Checked Cloud Console and MST and did not find anything called 'recents'. There is an 'overview' though. Would it be properly be called an "overview pane" or an "overview tab"? @Loquacity 

![image](https://user-images.githubusercontent.com/26616127/153905532-e2ae5bc4-bf3f-402f-8e3e-a6b4b54507d0.png)

Reply:
I think that's a Google thing, it's not related to our products. As for the second question, whoever takes on reorganising the Cloud guide will need to make some decisions there (probably in conjunction with the Design team).